json_test.go

```diff
@@ -15,17 +15,18 @@ func Test_ReadJSON_Success(t *testing.T) {
 	}
 
 	testfn := func(t *testing.T, filename string) {
-		t.Helper()
+		is := is.New(t)
+		is.Helper()
```

Comment:
Why is this a helper? seems inconsistent with filter_test.go.

Reply:
I don't know why this was a helper. No idea. I have remove it here and in the other json test. 🥪